app/assets/javascripts/channels/admin/repo_activity.js

```diff
@@ -0,0 +1,13 @@
+const RepoActivity = () => {
```

Comment:
**[eslint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
'RepoActivity' is assigned a value but never used. Allowed unused vars must match /^h$/. (no-unused-vars)